src/Features/Context/TiqrContext.php

```diff
@@ -17,6 +18,8 @@
 
 namespace App\Features\Context;
 
+require_once __DIR__ . '/../../../vendor/tiqr/tiqr-server-libphp/library/tiqr/Tiqr/OATH/OCRA.php';
```

Comment:
This class should already be autoloaded. If not, then please fix the issue at hand elsewhere. I believe we have a adapter file where certain lib-tiqr files are included.

See: https://github.com/OpenConext/Stepup-tiqr/blob/e20bdc3e66e310491bc3be824ee152f205206e79/src/Tiqr/TiqrFactory.php#L92
And: https://github.com/Tiqr/tiqr-server-libphp/blob/5416e1e2b9049f798f46a62fc22968828286c8ab/library/tiqr/Tiqr/AutoLoader.php